code/modules/clothing/glasses/glasses.dm

```diff
@@ -230,7 +230,7 @@ BLIND     // can't see anything
 
 /obj/item/clothing/glasses/safety/goggles/wasteland
 	name = "wasteland goggles"
-	desc = "A common pair of goggles, usually worn by Unathi who travel the wasteland. It's invaluable for keeping radioactive sand out of eyes."
+	desc = "A pair of old goggles common in the wasteland. Often worn by sailors and fishermen on the sea to keep saltwater out of their eyes before the war, now they take on new life to keep sand out of the eyes."
```

Comment:
A lot of the items here look pretty good. Just looking at the descriptions to be a little less winded;

"A pair of old goggles common in the Wasteland. A few denizens unfortunate enough to not keep this protection on them after the nukes dropped no longer have the ability to see."